src/Remark.jl

```diff
@@ -66,12 +77,12 @@ function _create_index_md(inputfile, outputdir; documenter = true)
 end
 
 function _create_index_html(outputdir, md_file, options = Dict(); title = "Title")
-    
+
     optionsjs = JSON.json(options)
     template = joinpath(_pkg_assets, "indextemplate.html")
     replacements = ["\$title" => title, "\$options" => optionsjs]
 
-    Base.open(joinpath(outputdir, "build", "index.html"), "w") do io
+    Base.open(joinpath(outputdir, "build", "index.html"), "a") do io
```

Comment:
Not sure I get this change. Why isn't it just `"w"` here?

Reply:
That is a mistake on my part.
I was testing something else and forgot to revert that